creation/web_base/singularity_lib.sh

```diff
@@ -1498,7 +1498,8 @@ singularity_locate_bin() {
     local test_out
     HAS_SINGULARITY=False
     local osg_singularity_binary="${OSG_SINGULARITY_BINARY:-${OSG_SINGULARITY_BINARY_DEFAULT}}"
-    local singularity_binary_override="${GLIDEIN_SINGULARITY_BINARY_OVERRIDE}"
+    local singularity_binary_override
+    singularity_binary_override="${GLIDEIN_SINGULARITY_BINARY_OVERRIDE:-$(gwms_from_config GLIDEIN_SINGULARITY_BINARY_OVERRIDE)}"
```

Comment:
I do not see any reference to `gwms_from_config`, so wanted to make sure if this is set/imported in a script that invokes the method that has this line?

Reply:
It's above, https://github.com/glideinWMS/glideinwms/blob/f0065cca512e12ef0a55ede42af390edf8b9cb3c/creation/web_base/singularity_lib.sh#L401